test/parser.jl

```diff
@@ -75,7 +80,7 @@ end
         y = MOI.addvariable!(model)
         MOI.set!(model, MOI.VariableName(), x, "x")
         MOI.set!(model, MOI.VariableName(), y, "y")
-        MOI.set!(model, MOI.ObjectiveFunction{MOI.ScalarAffineFunction{Float64}}(), MOI.ScalarAffineFunction(MOI.ScalarAffineTerm.([1.0, 2.0], [x, y]), 1.0))
+        MOI.set!(model, MOI.ObjectiveFunction{MOI.ScalarAffineFunction{Float64}}(), MOI.ScalarAffineFunction(MOI.ScalarAffineTerm.([1.0, -2.0], [x, y]), 1.0))
```

Comment:
I had to do this change because the tests were failing. No idea why it is not failing on master though

Reply:
Weird